homeassistant/components/tankerkoenig/config_flow.py

```diff
@@ -31,33 +38,27 @@
 
 
 async def async_get_nearby_stations(
-    hass: HomeAssistant, data: Mapping[str, Any]
-) -> dict[str, Any]:
+    tankerkoenig: Tankerkoenig, data: Mapping[str, Any]
+) -> list[Station]:
     """Fetch nearby stations."""
-    try:
-        return await hass.async_add_executor_job(
-            getNearbyStations,
-            data[CONF_API_KEY],
+    return await tankerkoenig.nearby_stations(
+        coordinates=(
             data[CONF_LOCATION][CONF_LATITUDE],
             data[CONF_LOCATION][CONF_LONGITUDE],
-            data[CONF_RADIUS],
-            "all",
-            "dist",
-        )
-    except customException as err:
-        return {"ok": False, "message": err, "exception": True}
+        ),
+        radius=data[CONF_RADIUS],
+        gas_type=GasType.ALL,
+        sort=Sort.DISTANCE,
+    )
 
 
 class FlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     """Handle a config flow."""
 
     VERSION = 1
 
-    def __init__(self) -> None:
-        """Init the FlowHandler."""
-        super().__init__()
-        self._data: dict[str, Any] = {}
-        self._stations: dict[str, str] = {}
+    _data: dict[str, Any] = {}
+    _stations: dict[str, str] = {}
```

Comment:
Please don't initialize class attributes to mutable objects unless explicitly required. It's very error prone. Use an init method to set instance attributes.